addons/mail/static/src/models/time/time.js

```diff
@@ -0,0 +1,56 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one2one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'Time',
+    identifyingFields: ['messaging'],
+    lifecycleHooks: {
+        _created() {
+            // TODO evrything should be done in a single setInterval and the field should be updated by comparing value. this.day !== current day.
+
+            // Update the minute field every minutes
+            this.intervalMinute = setInterval(
```

Comment:
`this.intervalMinute` => `this._everyMinuteIntervalId` _(more emphasis it's technical prop on record object rather than an actual field)_